scripts/generate/clean-up.ts

```diff
@@ -2,7 +2,7 @@ import fs from 'fs';
 import path from 'path';
 import {exec} from 'child_process';
 
-export function cleanUp() {
+export function cleanUp(type: string) {
```

Comment:
Not part of this PR obviously, but I wonder if the cleanup task should actually hard remove `node_modules` and reinstall them based on the "new" `package.json` to make sure only the relevant dependencies are installed.

Reply:
Agreed, I think right now you'd get a diff if you `yarn install`ed after committing our initial template project, which is not nice :/